pkg/controller/dcl/controller.go

```diff
@@ -472,7 +472,7 @@ func (r *Reconciler) handleDefaults(ctx context.Context, u *unstructured.Unstruc
 	for _, defaulter := range r.defaulters {
 		changed, err := defaulter.ApplyDefaults(ctx, k8s.ReconcilerTypeDCL, u)
 		if err != nil {
-			return fmt.Errorf("applying defaults: %w", err)
+			return fmt.Errorf("applying defaults in the dcl controller: %w", err)
```

Comment:
Yes, but for something that we're planning on cherry-picking we normally want to keep it minimal.  I think we'll get away with this one though.